plugins/browser-plugin-media-tracking/src/eventGroups.ts

```diff
@@ -12,6 +12,7 @@ export const AllEvents: EventGroup = [
 ];
 
 export const DefaultEvents: EventGroup = [
+  SnowplowEvent.READY,
```

Comment:
I also wonder if these new defaults are enough to warrant a v3.3 or if we think they are bug fixes as we should have been capturing a ready even all along? It doesn't quite feel significant enough for a v3.3 though but it's tempting as its a behaviour change and easier to recommend from a recommended tracker version for data modelling point of view.